src/sinks/util/unix.rs

```diff
@@ -201,7 +201,7 @@ impl Sink for UnixSink {
     }
 }
 
-#[cfg(all(feature = "tokio/uds", test))]
```

Comment:
I was also confused by this, but this style of `crate/feature` doesn't work in `cfg` annotations.